components/Button.js

```diff
@@ -0,0 +1,5 @@
+import styled from "styled-components";
+
+const Button = styled.button``;
+
+export { Button };
```

Comment:
I think it doesn't make sense at the moment but we will need it later in the future.

Reply:
To make it make sense already now, I added some styling. 😁 
 But yes, this component will actually only become important in the future. I think that it should then also become a functional component.